docs/how-to/enable_hugepages_for_upf.md

```diff
@@ -0,0 +1,68 @@
+# Enable HugePages for UPF
+
+## Requirements
+
+- A Kubernetes host with a CPU supporting AVX2 and RDRAND instructions (Intel Haswell, AMD Excavator or equivalent)
+- A Kubernetes cluster with the Multus addon enabled
+- Juju = 3.1/stable
+- HugePages is enabled in Kubernetes Host and it has enough 2Gi free HugePages 
+- Kubernetes Host CPU should have the flag "pdpe1gb" which allows to set 1Gi HugePages
```

Comment:
@dariofaccin Do you think it's enough to have this written as is or should we explicitly tell users how to do this?

Reply:
Thanks to Dario that helped me to find out the requirements while preparing them today.

Reply:
@gruyaume when discussing with Gulsum, we decided not to add specific information on how to enable huge pages as the procedure may depend on the OS (for example not using GRUB).